compose/.apps/duplicacy/duplicacy.labels.yml

```
@@ -0,0 +1,12 @@
+services:
+  duplicacy:
+    labels:
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.description: "Duplicacy is built on top of a new idea called Lock-Free Deduplication"
+      com.dockstarter.appinfo.nicename: "Duplicacy"
+      com.dockstarter.appvars.duplicacy_enabled: "false"
+      com.dockstarter.appvars.duplicacy_network_mode: ""
+      com.dockstarter.appvars.duplicacy_port_3875: "3875"
+      com.dockstarter.appvars.duplicacy_appdata_dir: "/home/$USER/.config/appdata"
+      com.dockstarter.appvars.duplicacy_docker_dir: "/home/$USER/.docker/"
+      com.dockstarter.appvars.duplicacy_backup_dir: "/home/$USER/backup"
```

```suggestion
      com.dockstarter.appvars.duplicacy_appdata_dir: "~/.config/appdata"
      com.dockstarter.appvars.duplicacy_docker_dir: "~/.docker/"
      com.dockstarter.appvars.duplicacy_backup_dir: "~/backup"
```
Can't use `$VARS` inside labels. DS will replace `~` automatically though.

---

I git pushed this before I did the Tdarr one lol.